python/tests/mlir/CMakeLists.txt

```diff
@@ -19,23 +19,26 @@ set(PYCUDAQMLIR_TEST_PARAMS
   pycudaqmlir_site_config=${CMAKE_CURRENT_BINARY_DIR}/lit.site.cfg.py)
 
 set(PYCUDAQMLIR_TEST_DEPENDS
-  _pycudaq _quakeDialect
+  CUDAQuantumPythonModules
   FileCheck)
 
-add_custom_target(pycudaq-mlir-test-depends DEPENDS ${PYCUDAQ_TEST_DEPENDS})
+add_custom_target(pycudaqmlir-test-depends DEPENDS ${PYCUDAQMLIR_TEST_DEPENDS})
+# set_target_properties(pycudaqmlir-test-depends PROPERTIES FOLDER "Tools")
+# set_target_properties(pycudaqmlir-test-depends PROPERTIES FOLDER "Tests")
 
-add_lit_testsuite(check-py-cudaq-mlir "Running cudaq python regression tests."
+add_lit_testsuite(check-pycudaq-mlir "Running cudaq python regression tests."
   ${CMAKE_CURRENT_BINARY_DIR}
-  PARAMS ${PYCUDAQ_TEST_PARAMS}
-  DEPENDS ${PYCUDAQ_TEST_DEPENDS}
+  PARAMS ${PYCUDAQMLIR_TEST_PARAMS}
+  DEPENDS ${PYCUDAQMLIR_TEST_DEPENDS}
 )
-set_target_properties(check-py-cudaq-mlir PROPERTIES FOLDER "Tools")
+set_target_properties(check-pycudaq-mlir PROPERTIES FOLDER "Tests")
+set_target_properties(check-pycudaq-mlir PROPERTIES FOLDER "Tools")
 
 add_lit_testsuites(PYCUDAQMLIR ${CMAKE_CURRENT_SOURCE_DIR}
-  PARAMS ${NVQPP_TEST_PARAMS}
-  DEPENDS ${NVQPP_TEST_DEPENDS})
+  PARAMS ${PYCUDAQMLIR_TEST_PARAMS}
+  DEPENDS ${PYCUDAQMLIR_TEST_DEPENDS})
 
 # Add nvqpp tests to the ctest suite
-add_test(NAME py-cudaq-tests-mlir
-  COMMAND ${CMAKE_COMMAND} --build ${CMAKE_BINARY_DIR} --target check-py-cudaq-mlir
-  WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
\ No newline at end of file
+add_test(NAME pycudaq-mlir
```

Comment:
- [ ] Confirm that this runs in CI...

Reply:
https://github.com/NVIDIA/cuda-quantum/actions/runs/8300601837/job/22719266576?pr=1405